config/manager/kustomization.yaml

```diff
@@ -13,4 +13,4 @@ kind: Kustomization
 images:
 - name: controller
   newName: quay.io/confidential-containers/operator
-  newTag: v0.10.0
+  newTag: v0.12.0
```

Comment:
could be also `latest` so that the kustomization in main HEAD always matches the latest image but `make bundle` seems to force a `VERSION` tag...

Reply:
Yeah, it looks like in the past I updated this back to latest post-release (https://github.com/confidential-containers/operator/commit/81a89f9fff01d5678043115ed40c43437d7adee6), so we could add this to our post release tasks? Or alternatively update `config/default/kustomization.yaml` to have
```
images:
- name: quay.io/confidential-containers/operator
  newTag: latest 
```
as I mentioned on slack, which I think should have the same effect?

Reply:
> Or alternatively update `config/default/kustomization.yaml` to have

+1

Reply:
Updating config/default/kustomization.yaml to use latest image is the best option IMO.

Reply:
Cool - we can do that once the release is all tied up I guess

Reply:
I took a quick look and it seems the overall flow needs rework: `make bundle` sets the image tag but it has no effect since `config/manifest` actually uses `config/release` as the target that sets an image tag that is decoupled from `VERSION` used by `make bundle`.